{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -27,6 +27,12 @@
 # Note: This key only used for development and testing.
 SECRET_KEY = env('DJANGO_SECRET_KEY', default='CHANGEME!!!')
 
+# SITE CONFIGURATION
+# ------------------------------------------------------------------------------
+# Hosts/domain names that are valid for this site
+# See https://docs.djangoproject.com/en/1.6/ref/settings/#allowed-hosts
```

Comment:
Why 1.6 version? Actually, the link leads to 404. What about https://docs.djangoproject.com/en/1.10/ref/settings/#allowed-hosts ?